### PR TITLE
Use https://repo.lightbend.com/helm-charts URL

### PR DESCRIPTION
**Pull request by marcoderama**
Also move `install-es.sh` into `enterprise-suite/scripts/` folder.

### CODE REVIEW

File: enterprise-suite/.helmignore [hunk 1]
```diff
@@ -25,3 +25,4 @@ README.md
 .idea/
 *.tmproj
 *~
+\#
```
Comment (marcoderama) [inline enterprise-suite/.helmignore]: This just makes sure temporary emacs files don't make it into our tarballs.

File: enterprise-suite/scripts/install-es.sh [hunk 2]
```diff
@@ -65,7 +65,7 @@ function import_credentials() {
 LIGHTBEND_COMMERCIAL_USERNAME=${LIGHTBEND_COMMERCIAL_USERNAME:-}
 LIGHTBEND_COMMERCIAL_PASSWORD=${LIGHTBEND_COMMERCIAL_PASSWORD:-}
 LIGHTBEND_COMMERCIAL_CREDENTIALS=${LIGHTBEND_COMMERCIAL_CREDENTIALS:-$HOME/.lightbend/commercial.credentials}
-ES_REPO=${ES_REPO:-https://lightbend.github.io/helm-charts}
+ES_REPO=${ES_REPO:-https://repo.lightbend.com/helm-charts}
```
[the remark this answers is not in the document]
Comment (jsravn) [reply enterprise-suite/scripts/install-es.sh]: Can you please include this change in the PR title / comment. As it's a big change.

[the remark this answers is not in the document]
Comment (jsravn) [reply enterprise-suite/scripts/install-es.sh]: Ok just went ahead and did it on my own :).

**Comment by jsravn** [comment]
We'll need to update es-docs as well after this change.

**Comment by jsravn** [comment]
Ok I updated es-docs, so no worries.

**Pull request merged**